src/pipeline/bundleFunctionalities.ts

```diff
@@ -0,0 +1,54 @@
+import { ifError } from "assert";
+import diprocheInterface from "../util/diproche/diprocheInterface";
+import { getAllIssues } from "../vocabularyChecker/detectWrongSyntax";
+
+function showVocabErrors(text: string) {
+		getAllIssues(text);
+}
+
+/**
+ * The following function should correct expressions like
+ * 2 += 2 4 in the user input to 2 + 2 = 4 or at least
+ * make the user aware of such mistakes
+ *
+ * function showWrongSyntax(text: string) {
+ *
+ * }
+ *
+ * The following function should convert a text like
+ * "It holds that 2 + 2 = 4"
+ * to "[It, holds, that, =[+[2,2],5]]"
+ *
+ * function convertToPrologList(text: string) {
+ *
+ * }
+ *
+ * The following function should tell the user which
+ * semantic mistakes he made. For example:
+ * The function should tell the user, that
+ * "2 + 2 = 5" is semantically wrong
+ *
+ * function showWrongSemantic(text: string) {
+ *
+ * }
+ *
+ */
+
+export function bundleEverything(text: string) {
+	ifError(showVocabErrors(text)); {
```

Comment:
The correct way in jest to test that something throws an error:
```typescript
expect(() => { 
  // your code
}).toThrow();
```
you don't need to open a block if you call just a single function: `expect(() => fubar()).toThrow();`. You can also negate your assertion: `expect(() => fubar()).not.toThrow();`. Even though it's type-correct, I'd **not** write `expect(fubar).toThrow();`. Javascript function-objects are weird, and Typescripts type-system for them is not helping much. My advise: Whenever passing around "function-pointers", do so with an explicit lambda-expression.